lms/djangoapps/courseware/date_summary.py

```diff
@@ -440,21 +456,12 @@ class CertificateAvailableDate(DateSummary):
     css_class = 'certificate-available-date'
     title = ugettext_lazy('Certificate Available')
 
-    @property
-    def active_certificates(self):
```

Comment:
I simply dropped this property in favor of `get_active_web_certificate` because it doesn't seem to be used by any other code anywhere. So no need to rewrite this logic here.